src/tracks/playabletrack/wavetrack/ui/WaveTrackAffordanceControls.cpp

```diff
@@ -220,6 +220,9 @@ unsigned WaveTrackAffordanceControls::KeyDown(wxKeyEvent& event, ViewInfo& viewI
     auto keyCode = event.GetKeyCode();
     switch (keyCode)
     {
+    case WXK_TAB:
+        SelectNextClip(viewInfo, project);
+        break;
```

Comment:
Tab key navigation didn't work when I tried it, but the simple missing piece, is a change of WaveTrackAffordanceControls::CaptureKey so it does not skip the event for tab key.

Reply:
Do we also want to implement leftward navigation among clips for Shift + TAB, as is done for labels?

Reply:
Fixed, tabbing works for me on Windows, but didn't test it on other platforms.
Also added a Shift+TAB, thanks